tf2onnx/utils.py

```diff
@@ -460,6 +460,10 @@ def get_max_value(np_dtype):
     return np.iinfo(np_dtype).max
 
 
+def get_min_value(np_dtype):
```

Comment:
looks like this function is not used in this PR?

Reply:
Ideally, we should use this function, but it will overflow when casting to low precision.
So in this PR, 1e9 and -1e9 are used for max and min value for index input (begin, end and strides).
Will re-enable it after we don't need to cater for the type restriction of Max op.